tests/output/ConsoleInteractionTest.py

```diff
@@ -560,24 +629,40 @@ def test_print_results_multiple_ranges(self):
                         affected_code)],
                 {abspath("some_file"): ["line " + str(i + 1) + "\n"
                                         for i in range(10)],
-                 abspath("another_file"): ["line " + str(i + 1) + "\n"
+                 abspath("another_file"): ["line " + str(i + 1)
                                            for i in range(10)]},
                 {},
                 color=False)
             self.assertEqual("""
 another_file
-|   1| line•1
+|   1| li{0}{1}
 
 another_file
-|   3| line•3
+|   3| li{0}{2}
 
 some_file
-|   5| line•5
-|   6| line•6
-|   7| line•7
+|   5| {3}
+|   6| {4}
+|   7| {5}
 |    | [NORMAL] ClangCloneDetectionBear:
-|    | Clone Found
-""",
+|    | {6}
+""".format(
+                highlight('ne', lexer, TerminalTrueColorFormatter(
+                    style=BackgroundSourceRange)).rstrip("\n"),
+                highlight(' 1', lexer, TerminalTrueColorFormatter()
+                          ).rstrip("\n"),
+                highlight(' 3', lexer, TerminalTrueColorFormatter()
+                          ).rstrip("\n"),
+                highlight('line 5', lexer, TerminalTrueColorFormatter(
+                    style=BackgroundSourceRange)).rstrip("\n"),
+                highlight('line 6', lexer, TerminalTrueColorFormatter(
+                    style=BackgroundSourceRange)).rstrip("\n"),
+                highlight('line 7', lexer, TerminalTrueColorFormatter(
+                    style=BackgroundSourceRange)).rstrip("\n"),
+                highlight("Clone Found", TextLexer(),
```

Comment:
`TextLexer` is for result messages. `lexer` is for `sourcerange`. The highlighter is different and also `TextLexer` doesn't need bullet marks for `spaces/tabs`. Hence the 2 different lexers.